fiftyone/operators/delegated.py

```diff
@@ -90,6 +90,8 @@ def set_running(self, doc_id, progress=None, run_link=None):
                 operation
             run_link (None): an optional link to orchestrator-specific
                 information about the operation
+            required_state (ExecutionRunState): an optional required state of the operation. If provided, the
```

Comment:
black will not reformat docstring but we will manually insert line breaks to keep to below 79